guides/rest_api.md

```diff
@@ -257,3 +289,62 @@ PUT /api/rest/v1/episodes/:id
 ```
 DELETE /api/rest/v1/episodes/:id
 ```
+
+## Audio
+
+### Parameters for Create & Update
+
+| Name                | Type      | Description                                                |
+| ------------------- | --------- | ---------------------------------------------------------- |
+| `audio[network_id]` | `integer` | Network ID. Either network ID or episode ID must be given. |
+| `audio[episode_id]` | `integer` | Episode ID. Either network ID or episode ID must be given. |
+| `audio[title]`      | `string`  |                                                            |
+| `audio[image]`      | `file`    | Audio image.                                               |
+
+### Create
+
+```
+POST /api/rest/v1/audios
+```
+
+### Read
+
+```
+GET /api/rest/v1/audios/:id
+```
+
+### Update
```

Comment:
As far as I read the rest stuff: Put should be used only if the entity is replaced completely, for field by field changes PATCH should be used. Since we rarely require the whole update, should we reference PATCH for all our changes instead of PUT?